### PR TITLE
[Jecoute][Candidate] Add Jecoute to candidate space + rename "J'écoute" to "Questionnaires"

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
[ch12858]

File: src/Controller/EnMarche/Jecoute/AbstractJecouteController.php [hunk 1]
```diff
@@ -69,6 +70,11 @@ public function jecouteSurveyCreateAction(
         SuggestedQuestionRepository $suggestedQuestionRepository,
         UserInterface $user
     ): Response {
+        if (JecouteSpaceEnum::CANDIDATE_SPACE === $this->getSpaceName()
```
Comment (ottaviano) [inline src/Controller/EnMarche/Jecoute/AbstractJecouteController.php:73]: what do you think about creating a function `checkCreateAccess` and use it in this generic createAction so we can override it in `JecouteCandidateController` for doing this specific job?
```
$this->checkCreateAccess();
```

File: templates/jecoute/surveys_list.html.twig [hunk 2]
```diff
@@ -55,14 +58,17 @@
...
                         {% set row = {
                             id: survey.id,
                             name: survey.name,
                             createdAt: survey.createdAt|date('d/m/Y'),
                             questionsCount: data.questions_count,
                             responsesCount: data.responses_count,
-                            zone: survey is instanceof('App\\Entity\\Jecoute\\LocalSurvey') ? survey.zone|trans : null,
+                            zone: survey is instanceof('App\\Entity\\Jecoute\\LocalSurvey')
+                                ? '<span class="adherent-dashboard-tag">'~('geo_zone.'~zone.type)|trans~'</span><br>'
+                                    ~"<div class=\"b__nudge--top-10\">#{zone.name} (#{zone.code})</span>",
```
Comment (ottaviano) [inline templates/jecoute/surveys_list.html.twig:71]: what will happen if the zone variable is null? is this possible?

**Review by ottaviano — APPROVED** [review]

File: templates/jecoute/surveys_list.html.twig (hunk 2 above)
Comment (illusionOfParadise) [reply templates/jecoute/surveys_list.html.twig:71]: For `LocalSurvey`, no.

Comment (ottaviano) [reply templates/jecoute/surveys_list.html.twig:71]: so this template displays only LocalSurvey? I ask for this code: `#{zone.name} (#{zone.code})` which does not work if zone is null

File: src/Controller/EnMarche/Jecoute/AbstractJecouteController.php (hunk 1 above)
Comment (ottaviano) [reply src/Controller/EnMarche/Jecoute/AbstractJecouteController.php]: @illusionOfParadise, could you override the `checkCreateAccess` in `JecouteCandidateController`, please?

**Pull request merged**